packages/cli/src/utils/config.js

```diff
@@ -0,0 +1,41 @@
+const { resolve } = require('path');
```

Comment:
i spent a little time re-organizing all of the functions in `utlis/*` into more specific files and then importing directly from them (instead of re-exporting them all, which messes up VSCode's "jump to definition" and function signature previews; I recognize this is a more me-specific problem, but a little spring cleaning is probably in order. `upload` probably doesn't belong in `utils/api.js`)